ansible/roles/status-report-install/tasks/main.yml

```diff
@@ -0,0 +1,77 @@
+---
+- name: Create application folder
+  file:
+    name: "/opt/status/src"
+    state: directory
+    owner: "{{ ansible_user }}"
+- name: Retrieve application source files
+  git:
+    clone: yes
+    force: yes
+    dest: "/tmp/status"
+    repo: "http://github.com/jimrigsbee/api.git"
```

Comment:
Should this point at jimrigsbee/api.git? Or is there something else to merge in here?

Reply:
@jimrigsbee

Reply:
Questions:
1. Should this be moved to a requirements.yml?
2. Should we use the personal repo as opposed to RedHatTraining?

Reply:
It is not a galaxy module so no it doesn't belong in requirements.yml.
If we use RedHatTraining, which we could, we have to introduce the use of an ssh key like we do in ansible-skylight to get the source code. Let's discuss as a team.

Reply:
TODO: create the `redhattraining/api.git` public repository and update line 12 to point to the new repo URL.